chain-impl-mockchain/Cargo.toml

```diff
@@ -29,6 +29,7 @@ custom_error = "1.6"
 
 [dev-dependencies]
 quickcheck = "0.8"
+quickcheck_macros = "0.8.0"
```

Comment:
why do you need this one now?

Reply:
We don't need it, but it makes our lives a bit easier. Macro by example prevents rustfmt from formatting test bodies unlike proc macro. This dependency is low cost too, it itself doesn't have any dependencies not covered by serde_derive anyway.